easybuild/easyconfigs/p/PyTorch/PyTorch-1.0.1-fosscuda-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,240 @@
+source_urls = ['https://github.com/pytorch/pytorch/archive']
+# PyTorch pulls in a bunch of submodules which don't have releases.
+# We download the submodule revisions from their repos.
+# determine commit of additional sources via https://github.com/pytorch/pytorch/tree/v1.0.1/third_party
+sources = [
+    'v%(version)s.tar.gz',  # PyTorch
+    {
+        'source_urls': ['https://github.com/facebookincubator/gloo/archive'],
+        'download_filename': '670b4d4aa46886cc66874e2a4dc846f5cfc2a285.tar.gz',
+        'filename': 'gloo-20190201.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/gloo'),
+    },
+    {
+        'source_urls': ['https://github.com/google/googletest/archive'],
+        'download_filename': '2fe3bd994b3189899d93f1d5a881e725e046fdc2.tar.gz',
+        'filename': 'googletest-20180831.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/googletest'),
+    },
+    {
+        'source_urls': ['https://github.com/pybind/pybind11/archive'],
+        'download_filename': '5c8746ff135abb390bf95944be593e895a586a50.tar.gz',
+        'filename': 'pybind11-20180911.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/pybind11'),
+    },
+    {
+        'source_urls': ['https://github.com/wjakob/clang-cindex-python3/archive'],
+        'download_filename': '6a00cbc4a9b8e68b71caf7f774b3f9c753ae84d5.tar.gz',
+        'filename': 'clang-cindex-python3-20170330.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/pybind11/tools/clang'),
+    },
+    {
+        'source_urls': ['https://github.com/pytorch/cpuinfo/archive'],
+        'download_filename': '89fe1695edf9ee14c22f815f24bac45577a4f135.tar.gz',
+        'filename': 'cpuinfo-20190117.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/cpuinfo'),
+    },
+    {
+        'source_urls': ['https://github.com/onnx/onnx/archive'],
+        'download_filename': '42804705bdbf179d1a98394008417e1392013547.tar.gz',
+        'filename': 'onnx-20181204.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/onnx'),
+    },
+    {
+        'source_urls': ['https://github.com/google/protobuf/archive'],
+        'download_filename': '2761122b810fe8861004ae785cc3ab39f384d342.tar.gz',
+        'filename': 'protobuf-20171113.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/protobuf'),
+    },
+    {
+        'source_urls': ['https://github.com/Maratyszcza/pthreadpool/archive'],
+        'download_filename': '13da0b4c21d17f94150713366420baaf1b5a46f4.tar.gz',
+        'filename': 'pthreadpool-20181008.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (pytorchdir, 'third_party/pthreadpool'),
+    },
+    {
+        'source_urls': ['https://github.com/Maratyszcza/FXdiv/archive'],
+        'download_filename': 'b742d1143724d646cd0f914646f1240eacf5bd73.tar.gz',
+        'filename': 'FXdiv-20181016.tar.gz',
```

Comment:
This should probably be FXdiv-20181116.tar.gz, that's what the commit says anyway.
Should be change in the pure foss version too.

Reply:
done